src/useStyles.js

```diff
@@ -25,7 +25,7 @@ function useStyles(...styles) {
     }
   }
   if (isBrowser) {
-    useEffect(runEffect, [])
+    useLayoutEffect(runEffect, styles)
```

Comment:
what are you trying to solve here?
I know this repo does not have any tests, but we should start adding them in. Would be nice if you can provide some for your special usecase